src/BBT.StructureTools.Extensions/BBT.StructureTools.Extensions.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Copyright>Copyright © BBT Software AG</Copyright>
+    <Description>Extensions to BBT.StructureTools (e.g. implementations of ICreateConvertHelper and IConvertHelperFactory interfaces)</Description>
+    <PackageIcon></PackageIcon>
+    <RepositoryUrl>https://github.com/bbtsoftware/BBT.StructureTools.git</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageTags>copy compare convert</PackageTags>
+    <PackageReleaseNotes>https://github.com/bbtsoftware/BBT.StructureTools/releases/tag/1.0.4</PackageReleaseNotes>
+  </PropertyGroup>
```

Comment:
Have these properties the same as [here](src/BBT.StructureTools.Extensions/Convert/CreateTargetConvertTarget/ICreateTargetConvertTargetHelper.cs)

Reply:
Done.